src/handlers/check_commits/force_push_range_diff.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Context as _;
+
+use crate::config::RangeDiffConfig;
+use crate::github::GithubCompare;
+use crate::github::IssueRepository;
+use crate::github::IssuesEvent;
+use crate::handlers::Context;
+
+pub(super) async fn handle_event(
+    ctx: &Context,
+    host: &str,
+    _config: &RangeDiffConfig,
+    event: &IssuesEvent,
+    compare_after: &GithubCompare,
+) -> anyhow::Result<()> {
+    if !matches!(event.action, crate::github::IssuesAction::Synchronize) {
+        return Ok(());
+    }
+
+    let (Some(before), Some(after)) = (event.before.as_ref(), event.after.as_ref()) else {
+        tracing::warn!("synchronize event but no before or after field");
+        return Ok(());
+    };
+
+    let org = event.repository.owner();
+    let repo = event.repository.name();
+
+    let compare_before = ctx
+        .github
+        .compare(
+            &IssueRepository {
+                organization: org.to_string(),
+                repository: repo.to_string(),
+            },
+            &event.issue.base.as_ref().context("no base ref")?.git_ref,
```

Comment:
Why does `Issue::compare` use `base.sha`, and this uses `.git_ref`? I guess it doesn't matter that much, but it seems inconsistent/confusing.

Reply:
Hum, mistake on my part, switch to `base.sha`.